ethereum/spec/src/main/java/tech/pegasys/teku/spec/schemas/SchemaDefinitionsCapella.java

```diff
@@ -61,6 +63,10 @@ public class SchemaDefinitionsCapella extends SchemaDefinitionsBellatrix {
   private final BuilderBidSchema builderBidSchemaCapella;
   private final SignedBuilderBidSchema signedBuilderBidSchemaCapella;
 
+  private final HistoricalSummary.HistoricalSummarySchema historicalSummarySchema;
+
+  private final SszListSchema<HistoricalSummary, ?> historicalSummariesSchema;
```

Comment:
Does this belong here? It looks like it should be accessed as part of `BeaconStateSchemaCapella`.